### PR TITLE
Add how to solve pipenv error to readme

### PR DESCRIPTION
**Pull request by ki4070ma**
For https://github.com/appium/python-client/issues/400

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,8 +55,9 @@ download and unarchive the source tarball (Appium-Python-Client-X.X.tar.gz).
     - It generates readable changelog
 - Setup
     - `pip install --user pipenv`
-    - `pipenv lock --clear`
-    - `pipenv install --dev --system`
+    - `python -m pipenv lock --clear`
+       - If you have [this error](https://github.com/pypa/pipenv/issues/187https://github.com/pypa/pipenv/issues/187), refer the link to solve.
+    - `pytonh -m pipenv install --dev --system`
```
Comment (mykola-mokhnach) [inline README.md]: pytonh?

File: README.md [hunk 2]
```diff
@@ -55,8 +55,9 @@ download and unarchive the source tarball (Appium-Python-Client-X.X.tar.gz).
     - It generates readable changelog
 - Setup
     - `pip install --user pipenv`
-    - `pipenv lock --clear`
-    - `pipenv install --dev --system`
+    - `python -m pipenv lock --clear`
+       - If you have [this error](https://github.com/pypa/pipenv/issues/187https://github.com/pypa/pipenv/issues/187), refer the link to solve.
```
Comment (mykola-mokhnach) [inline README.md]: I'd rather explicitly write what the error message is

Comment (mykola-mokhnach) [inline README.md]: doubledouble paste?

File: README.md (hunk 1 above)
Comment (ki4070ma) [reply README.md]: ...Thanks and sorry

File: README.md (hunk 2 above)
Comment (ki4070ma) [reply README.md]: ...Thanks and sorry2

File: README.md [hunk 3]
```diff
@@ -55,8 +55,28 @@ download and unarchive the source tarball (Appium-Python-Client-X.X.tar.gz).
     - It generates readable changelog
 - Setup
     - `pip install --user pipenv`
-    - `pipenv lock --clear`
-    - `pipenv install --dev --system`
+    - `python -m pipenv lock --clear`
+       - If you have below error, refer [this link]((https://github.com/pypa/pipenv/issues/187)) to solve it.
```
Comment (ki4070ma) [inline README.md]: (( -> (

File: README.md [hunk 4]
```diff
@@ -55,8 +55,28 @@ download and unarchive the source tarball (Appium-Python-Client-X.X.tar.gz).
     - It generates readable changelog
 - Setup
     - `pip install --user pipenv`
-    - `pipenv lock --clear`
-    - `pipenv install --dev --system`
+    - `python -m pipenv lock --clear`
+       - If you have below error, refer [this link](https://github.com/pypa/pipenv/issues/187) to solve it.
+          ```
+          Locking [dev-packages] dependencies…
```
Comment (ki4070ma) [inline README.md]: Too much? @mykola-mokhnach

Comment (mykola-mokhnach) [reply README.md]: yeah, it would be enough to just keep the error message itself: `Locking Failed! unknown locale: UTF-8`

File: README.md [hunk 5]
```diff
@@ -55,8 +55,28 @@ download and unarchive the source tarball (Appium-Python-Client-X.X.tar.gz).
     - It generates readable changelog
 - Setup
     - `pip install --user pipenv`
-    - `pipenv lock --clear`
-    - `pipenv install --dev --system`
+    - `python -m pipenv lock --clear`
+       - If you have below error, refer [this link](https://github.com/pypa/pipenv/issues/187) to solve it.
```
Comment (mykola-mokhnach) [inline README.md]: if you experience the below error then refer https://github.com/pypa/pipenv/issues/187 to solve it

**Comment by KazuCocoa** [comment]
https://github.com/appium/python-client/issues/400#issuecomment-502420851

File: README.md (hunk 5 above)
Comment (ki4070ma) [reply README.md]: Thanks!

File: README.md (hunk 4 above)
Comment (ki4070ma) [reply README.md]: Thanks, it's smart.

**Comment by ki4070ma** [comment]
TODO: Remove flaky test(func_test_android2) until resolved.

**Pull request merged**